model_compression_toolkit/data_generation/common/enums.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from enum import Enum
+
+
+class EnumBaseClass(Enum):
+    @classmethod
+    def get_values(cls):
+        """
+        Get the list of values corresponding to the enum members.
+
+        Returns:
+            List of values.
+        """
+        return [mem.value for mem, mem in cls.__members__.items()]
```

Comment:
Isn't `mem, mem` overrides the first "mem" variable?
Why not:
`[mem.value for mem in cls]`
(I don't know if it would work but it worth a shoot)